prov/verbs/src/fi_verbs.c

```diff
@@ -64,6 +64,13 @@ struct fi_ibv_gl_data fi_ibv_gl_data = {
 		.use_name_server	= 1,
 		.name_server_port	= 5678,
 	},
+
+	.msg			= {
+		/* Disabled by default. Use XRC transport for message
+		 * endpoint only if it is explicitly requested */
+		.use_xrc		= 0,
+		.xrcd_filename		= "/tmp/verbs_xrcd",
```

Comment:
If you squint and turn your head, and xrcd has elements of being related to a shared AV.  I wonder if it wouldn't make sense for the xrcd_filename to come from the rxm::av_attr::name.  This would be an area where rxm would need to pass data down specifically for the xrc implementation, maybe through a provider specific control function on the verbs domain object?

Reply:
Can this be deferred for future consideration?

Reply:
yes - thanks